### PR TITLE
Shrink volume size for aws and azure to 1tb for 8-core machines

### PR DESCRIPTION
**Pull request by ZhouXing19**

### CODE REVIEW

File: cloudDetails/azure.json [hunk 1]
```diff
@@ -109,35 +151,35 @@
       "Standard_E32as_v4": null,
       "Standard_E32as_v5": {
         "roachprodArgs": {
+          "azure-volume-size": "1000",
```
Comment (xun-cockroachlabs) [inline cloudDetails/azure.json:154]: We should keep 2.5tb for this one, since it's vcpu32 type.

Comment (xun-cockroachlabs) [inline cloudDetails/azure.json:154]: We should keep 2.5tb for this one, since it's vcpu32 type.

File: cloudDetails/azure.json [hunk 2]
```diff
@@ -146,13 +188,30 @@
       "Standard_E32s_v4": null,
       "Standard_E32s_v5": {
         "roachprodArgs": {
+          "azure-volume-size": "1000",
```
Comment (xun-cockroachlabs) [inline cloudDetails/azure.json:191]: same here, keep 2.5tb.

File: cloudDetails/azure.json (hunk 1 above)
Comment (ZhouXing19) [reply cloudDetails/azure.json]: Thank you for pointing it out! Deleted this line.

File: cloudDetails/azure.json (hunk 2 above)
Comment (ZhouXing19) [reply cloudDetails/azure.json]: Done.

**Comment by ZhouXing19** [comment]
@xun-cockroachlabs thanks! removed those two lines.

**Comment by xun-cockroachlabs** [comment]
lgtm

**Comment by ZhouXing19** [comment]
TFTR!

**Pull request merged**